adr/recipes/try_users.py

```diff
@@ -11,30 +11,39 @@
 
 from collections import defaultdict
 
-from ..recipe import RecipeParser
-from ..query import run_query
+from ..recipe import RecipeParser, execute_query
 
 
-def run(args, config):
-    parser = RecipeParser('date')
-    parser.add_argument('--limit', type=int, default=25,
-                        help="Maximum number of users to return")
-    parser.add_argument('--sort-key', type=int, default=1,
-                        help="Key to sort on (int, 0-based index)")
-    args = parser.parse_args(args)
+def get_run_contexts():
```

Comment:
Could we change this function to a global `RECIPE_CONTEXT` variable instead?

Reply:
Oh I misunderstood it. Yes I can change it into a global variable. Thanks :-)